omnibus/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
-gem "omnibus", git: "https://github.com/chef/omnibus", branch: "master"
-gem "omnibus-software", git: "https://github.com/chef/omnibus-software", branch: "master"
+gem "omnibus", git: "https://github.com/chef/omnibus", branch: "jm/deep_sign"
```

Comment:
If we're waiting on other PR's will this get updated to point to master at that point? If so, please request re-review at that time. Thanks =)

Reply:
We may look at merging based off the branches to get builds in current. We'll leave inspect til last and if we get everything else in order I'll update to point at master. I just wanted prs open so folks are aware of the work going on.